src/_P039_Thermosensors.ino

```diff
@@ -726,9 +746,15 @@ float readMax6675(struct EventStruct *event)
 {
   uint8_t CS_pin_no = get_SPI_CS_Pin(event);
 
-  // "transfer" 0x0 and read the Data from the Chip
-  // uint16_t rawvalue = SPI.transfer16(0x0);
-  uint16_t rawvalue = read16BitRegister(CS_pin_no, 0x0000);
+  uint8_t messageBuffer[2] = {0x00, 0x00};
```

Comment:
Just a small trick.
Initializing to zero values (and only to zero values, not other values) can be done like this:
```c++
uint8_t messageBuffer[2] = {0};
```
It won't make a lot of difference on this array holding 2 elements, but you can imagine the gain on much larger arrays :)

Reply:
Solved in upcoming push !